docs/changelog.md

```diff
@@ -131,6 +131,18 @@ Therefore, you should upgrade to Cassandra 3 or higher.
     or never changed the value of `storage.cql.storage-compact=true`, during the 
     upgrade process you have to ensure your data is correctly migrated.
 
+##### Replace the Gremlin Server startup class with JanusGraph variant
+
+`JanusGraphServer` class allows us to add JanusGraph specfic settings to the 
+`gremlin-server.yaml` or `janusgraph-server.yaml`. Further, JanusGraph will ensure 
+serializers are configured as expected for running JanusGraph and interact with Gremlin. 
```

Comment:
I think it would be good state here explicitly which elements are now configured automatically by default to make it easier for users to update their config. The new default config could also be linked as an additional help.